### PR TITLE
updated it.js

### PR DESCRIPTION
**Pull request by mxtteismad**
updated it.js for version 4.0.9 by mxtteismad (mxtteismad#5209)

### CODE REVIEW

**Comment by mty22** [comment]
Nice. Thank you!

I've notified @MrGira on Discord to review.

@mxtteismad can you please comment with your Krunker IGN and Discord ID.

File: it.js [hunk 1]
```diff
@@ -1,821 +1,821 @@
...
+settings.controls.change Associazione tasti
+settings.controls.press Premi un tasto qualsiasi
+settings.local.header Localizzazione <span id="requiresRestart"><span style="color: #eb5656">*</span> necessita un riavvio</span>
+settings.local.region Regione predefinita
+settings.local.lang Lingua
+settings.network.header Rete
+settings.network.sendRate Tickrate alto
+settings.network.netRerouting Instradamento automatico (Influisce sul Ping)
+settings.network.lagComp Compensazione Lag
+settings.network.hitReg Test della registrazione del colpo
```
Comment (MrGira) [inline it.js:94]: I would suggest the form "Test registrazione dei colpi"

File: it.js [hunk 2]
```diff
@@ -1,821 +1,821 @@
...
+windows.controls.chat Chat
+windows.controls.voice Chat Vocale
+windows.controls.list Lista giocatori
+windows.controls.board Classifica
+windows.controls.interact Interagisci
+windows.controls.interactSec Interazione secondaria
+windows.controls.confirm Conferma interazione
+windows.controls.drop Lascia
+windows.controls.wepVis Aziona visibilità arma
+windows.controls.kickVoteY Vota per kickare [Si]
```
Comment (MrGira) [inline it.js:470]: Change "Si" with "Sì".

Comment (MrGira) [reply it.js:470]: Would be best to translate "Kickare" with "Vota per espellere"

File: it.js [hunk 3]
```diff
@@ -1,821 +1,821 @@
...
+windows.controls.kickVoteN Vota per kickare [No]
+windows.controls.kpdVoteY Voto KPD [Si]
+windows.controls.kpdVoteN Voto KPD [No]
+windows.controls.specFree attiva/disattiva cam libera
```
Comment (MrGira) [inline it.js:474]: Change "Attiva/disattiva" with "Aziona".

File: it.js [hunk 4]
```diff
@@ -1,821 +1,821 @@
...
+windows.mods.community.none Nessuna mod trovata
+windows.publish.mod.header Pubblica Mod
+windows.publish.mod.thumbnail Nessuna copertina
+windows.publish.mod.name Nome Mod
+windows.publish.mod.paste URL Mod
+windows.publish.mod.update Pubblica/Aggiorna
+windows.publish.mod.login per creare e aggiornare Mod!
+windows.secondary.header Scegli secondaria
+windows.secondary.unlocked Sbloccate
+windows.secondary.customOnly Solo custom
```
Comment (MrGira) [inline it.js:581]: Change "custom" with "personalizzate".

File: it.js [hunk 5]
```diff
@@ -1,821 +1,821 @@
...
+server.config.kCams Camera uccisione
+server.config.noReload Niente ricarica
+server.config.private Privata
+server.config.krRewards Premi KR
+server.config.forceDefault Forza impostazioni predefinite
+server.config.warmupTime Tempo riscaldamento
+server.config.gamRounds Rounds
+server.config.objtvTime Tempo obiettivo
+server.config.forceSpawn Tempo rinascita automatica
+server.config.intermTmr Pausa (s)
```
Comment (MrGira) [inline it.js:676]: The correct translation would be "Intervallo".

File: it.js [hunk 6]
```diff
@@ -1,821 +1,821 @@
...
+server.config.hitBoxPad Dimensione Hitbox
+server.config.hpRegen Rigenerazione vita
+server.config.disableB Disabilita bordi
+server.config.throwMel Armi da mischia lanciabili
+server.config.noCosm Disabilita cosmetici
+server.config.tstCmp Modalità test
+server.config.limitClasses Limita classi
+server.config.tmSize Dimensione squadra
+server.config.fallDmg Danno da caduta
+server.config.fallDmgThr Taglio del danno da caduta
```
Comment (MrGira) [inline it.js:690]: "Taglio" is incorrect, the correct form is "Soglia danno da caduta".

File: it.js [hunk 7]
```diff
@@ -1,821 +1,821 @@
...
+server.config.hpRegen Rigenerazione vita
+server.config.disableB Disabilita bordi
+server.config.throwMel Armi da mischia lanciabili
+server.config.noCosm Disabilita cosmetici
+server.config.tstCmp Modalità test
+server.config.limitClasses Limita classi
+server.config.tmSize Dimensione squadra
+server.config.fallDmg Danno da caduta
+server.config.fallDmgThr Taglio del danno da caduta 
+server.config.noDraws Nessun disegno
```
Comment (MrGira) [inline it.js:691]: "Disegno" is incorrect, the "draw" is "pareggio". "Senza pareggi" or "Nessun pareggio" would be best.

**Review by MrGira — CHANGES_REQUESTED** [review]
@mxtteismad there are some inaccuracies with the translation. 
If you have some doubt's or needs for clarification, don't be afraid to ping me here or on Krunker Bunker's tranlator chat.

**Review by MrGira — APPROVED** [review]
Ok, looks good.
Next time, if you have difficulties understanding the context of a line, feel free to contact me.

**Comment by mty22** [comment]
kr queued

thank you

**Pull request merged**